java/connector-node/risingwave-sink-jdbc/src/main/java/com/risingwave/connector/JDBCSink.java

```diff
@@ -199,6 +207,47 @@ private PreparedStatement prepareStatement(SinkRow row) {
         }
     }
 
+    private PreparedStatement generatePreparedStatement(
+            String inputStmt, SinkRow row, Object[] updateDeleteValueBuffer) throws SQLException {
+        PreparedStatement stmt = conn.prepareStatement(inputStmt, Statement.RETURN_GENERATED_KEYS);
+        var columnNames = getTableSchema().getColumnNames();
+        int placeholderIdx = 1;
+        for (int i = 0; i < row.size(); i++) {
+            switch (getTableSchema().getColumnType(columnNames[i])) {
+                case INTERVAL:
+                    if (targetDbType == DatabaseType.POSTGRES) {
+                        stmt.setObject(placeholderIdx++, new PGInterval((String) row.get(i)));
+                    } else {
+                        stmt.setObject(placeholderIdx++, row.get(i));
+                    }
+                    break;
+                case JSONB:
```

Comment:
Can we have a unit test that uses the json payload to pass the interval, jsonb and bytea value and sink to a PostgreSQL instance? In this way we can test whether the current implementation works for both MySQL and PostgreSQL in a lighter way by simply running the unit test with maven instead of launching the whole cluster and running a set of commands.

Reply:
For unit test maybe we can just use the `ArraySinkRow` to construct rows easily.